### PR TITLE
NanoUI Goes To Brazil

### PR DESCRIPTION
**Pull request by ShadowLarkens**
And may ye burn in hell, NanoUI.

This converts every last remaining template to TGUI, and nukes NanoUI entirely.

UIs Converted:
 - Accounts Terminal
 - Agent Card
 - Anesthetic Pump
 - Arcade Battle
 - Battery Rack
 - Fax Machine
 - Guest Pass
 - Janicart
 - pAI Interface
 - Photocopier
 - portable turrets
 - Teleporter
 - Telescience
 - Turbine

Accounts Terminal
Agent Card
Anesthetic Pump
Arcade Battle
Battery Rack (all issues already fixed, didn't feel like setting the fucker up again)
Fax Machine
Guest Pass
Janicart
pAI Interface
Photocopier
portable turrets
Teleporter
Telescience
Turbine

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (KasparoVy) [inline .vscode/settings.json:1]: Wots dis

File: code/controllers/subsystems/persistence.dm [hunk 2]
```diff
@@ -31,7 +31,7 @@ SUBSYSTEM_DEF(persistence)
 		return
 
 //	if((!T.z in GLOB.using_map.station_levels) || !initialized)
-	if(!T.z in using_map.station_levels)
+	if(!(T.z in using_map.station_levels))
```
Comment (KasparoVy) [inline code/controllers/subsystems/persistence.dm:34]: Nice catch

**Comment by KasparoVy** [comment]
Fuck that was fast. Awesome job

File: .vscode/settings.json (hunk 1 above)
Comment (ShadowLarkens) [reply .vscode/settings.json:1]: whoops

**Pull request merged**